snyk.go

```diff
@@ -127,8 +127,9 @@ type projectOrg struct {
 }
 
 type project struct {
-	Name string `json:"name,omitempty"`
-	ID   string `json:"id,omitempty"`
+	Name          string `json:"name,omitempty"`
+	ID            string `json:"id,omitempty"`
+	IsMonitored   bool `json:"isMonitored,omitempty"`
```

Comment:
Small formatting nit 😉 

```suggestion
	ID            string `json:"id,omitempty"`
	IsMonitored   bool   `json:"isMonitored,omitempty"`
```